source/agora/flash/api/FlashAPI.d

```diff
@@ -175,6 +175,12 @@ public struct ChannelUpdate
             && this.fixed_fee == other.fixed_fee
             && this.proportional_fee == other.proportional_fee;
     }
+
+    /// Required when `opEquals` is implemented
+    public size_t toHash () const scope @trusted pure nothrow @nogc
+    {
+        return *(cast(size_t*) this.chan_id[].ptr);
```

Comment:
Could use a `static assert(sizeof(this.chan_id) >= sizeof(size_t))`

Reply:
Done